tools/barista/src/main.ts

```diff
@@ -64,14 +63,17 @@ function createInternalLinksTransformer(): BaPageTransformer {
 async function createExampleInlineSourcesTransformer(): Promise<
   BaPageTransformer
 > {
-  if (!existsSync(EXAMPLES_METADATA)) {
+  if (!existsSync(environment.examplesMetadataDir)) {
     throw new Error(
-      '"examples-metadata.json" not found. Make sure to run "examples-tools" first.',
+      `"${environment.examplesMetadataFileName}" not found. Make sure to run "examples-tools" first.`,
     );
   }
-  const examplesMetadata = (await fs.readFile(EXAMPLES_METADATA, {
-    encoding: 'utf8',
-  })).toString();
+  const examplesMetadata = (await fs.readFile(
```

Comment:
please try catch if file exists (and try the encoding trick I mentioned above)

Reply:
In this case there's already an exists-check (line 66) and also `encoding: 'utf8'` is used in line 74. ;)

Reply:
line 76 is not necessary then

Reply:
true 😄